README.md

```diff
@@ -38,6 +38,7 @@ For all contributors, please be sure to read the [Contributing](docs/Contributin
 - [`<Badge />`](docs/Badge.md)
 - [`<Banner />`](docs/Banner.md)
 - [`<Box />`](docs/Box.md)
+- [`<BlockLink />`](docs/BlockLink.md)
```

Comment:
I feel like we haven't been good about keeping this up to date, and we could probably ditch this list once the docs site is live

Reply:
Yeah, reviewed the docs site PR as well and it looks great. This list docs on the main README can be ditched for easier doc maintenance.